app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -19,9 +19,9 @@ object FeatureFlags {
     val pullToRefreshEnabled = Config.channel.isNightlyOrDebug
 
     /**
-     * Enables the Addresses autofill feature.
+     * Enables the Sync Addresses feature.
      */
-    val addressesFeature = Config.channel.isNightlyOrDebug
+    const val syncAddressesFeature = false
```

Comment:
should this stay as "Config.channel.isNightlyOrDebug" or is it the syncing _specifically_ that still isn't enabled?

Reply:
Syncing is still specifically not enabled.